### ISSUE TITLE
Improve emphasis of keyboard shortcuts

### ISSUE THREAD

**Issue by ronso0** [mixxxdj/manual#145]
IMO they should stand out more than they do now:
![image](https://user-images.githubusercontent.com/5934199/95659544-1855c580-0b22-11eb-8180-727ffacbd2ec.png)

**Comment by hannydevelop** [comment]
I will work on this now @ronso0

**Comment by ronso0** [comment]
Before actually commiting changes it would be nice if you'd show some possible variations so we can discuss what fits best.

**Comment by hannydevelop** [comment]
> Before actually commiting changes it would be nice if you'd show some possible variations so we can discuss what fits best.

ok! i would do that

**Comment by hannydevelop** [comment]
> Before actually commiting changes it would be nice if you'd show some possible variations so we can discuss what fits best.

This is an example of a shortcut key as written in manual-2.2.x

* Activate the search input field by clicking in it or pressing :kbd:`CTRL` +
  :kbd:`F` (Windows/Linux) or :kbd:`CMD` + :kbd:`F` (Mac).

These are possible suggestions to add emphasis to the keys. An image is attached to show what
each empahasis looks like after build respectively:

* Activate the search input field by clicking in it or pressing :guilabel:`CTRL` +
  :guilabel:`F` (Windows/Linux) or :guilabel:`CMD` + :guilabel:`F` (Mac).

* Activate the search input field by clicking in it or pressing **CTRL** +
  **F** (Windows/Linux) or **CMD** + **F** (Mac).

* Activate the search input field by clicking in it or pressing `CTRL` +
  `F` (Windows/Linux) or `CMD` + `F` (Mac).

* Activate the search input field by clicking in it or pressing :file:`CTRL` +
  :file:`F` (Windows/Linux) or :file:`CMD` + :file:`F` (Mac).

I think the first emphasizes more, Which do you think emphasizes the hot keys more? 
![Capture1](https://user-images.githubusercontent.com/62933980/95912746-1e0a1000-0d9b-11eb-9ce5-c513dc5b9441.JPG)

**Comment by ronso0** [comment]
First impression: 
* we may use :file: but with black font color
* maybe simple bold text is already sufficient

To test another bold variant, could you please also post an image where you replaced the strings
CTRL > Ctrl
CMD > Cmd (or even ⌘)

**Comment by Holzhaus** [comment]
> * we may use :file: but with black font color

We should definitely keep using `:kbd:`. Using something else is plain wrong, because it changes the semantic meaning. If we want to modify how it looks, we can improve the `sources/_static/css/mixxx.css` file. I like the `:guilabel:` style best, because it kind of looks like an actual key, but if you prefer the style from :file: is also okay :wink:

**Comment by ronso0** [comment]
Yes, we talk about the styles only which should be applied to :kbd:

The rounded corners of :guilabel: are nice indeed, but :guilabel: itself is too big IMO. Let's see how it looks with Capitals only

**Comment by Holzhaus** [comment]
We could reduce the padding and the font size a bit.

**Comment by hannydevelop** [comment]
> Yes, we talk about the styles only which should be applied to :kbd:
> 
> The rounded corners of :guilabel: are nice indeed, but :guilabel: itself is too big IMO. Let's see how it looks with Capitals only

So, @Holzhaus and @ronso0 we would still be going with ```:kbd:``` but tweaking the styles a little?

**Comment by hannydevelop** [comment]
> First impression:
> 
> * we may use :file: but with black font color
> * maybe simple bold text is already sufficient
> 
> To test another bold variant, could you please also post an image where you replaced the strings
> CTRL > Ctrl
> CMD > Cmd (or even ⌘)

Hello @ronso0 should i still go ahead with testing another bold variant?

**Comment by ronso0** [comment]
Hm, it looks like we're focused on the mix of :guilabel: and :file:
* round corners
* less padding & a little smaller font than :guilabel:
* bold
* Capitals only (CTRL > Ctrl, COMMAND > Cmd / ⌘)

Yes, it would be nice if you could create a preview for that.

**Comment by hannydevelop** [comment]
> Hm, it looks like we're focused on the mix of :guilabel: and :file:
> 
> * round corners
> * less padding & a little smaller font than :guilabel:
> * bold
> * Capitals only (CTRL > Ctrl, COMMAND > Cmd / ⌘)
> 
> Yes, it would be nice if you could create a preview for that.

ok, I would work on that!

**Comment by ronso0** [comment]
I haven't worked on manual styles myself so far, and I didn't find any definition for :kb: in the static stylesheet, so I guess you need to look how :guilabel: is styled and create an overriding rule in https://github.com/mixxxdj/manual/blob/manual-2.2.x/source/_static/css/mixxx.css
Any other hints on that @Holzhaus ?

**Comment by Holzhaus** [comment]
Nope, inspect the kbd element and check what CSS class it has. Then override that class via mixxx.css just as you described.

**Comment by hannydevelop** [comment]
Hello @ronso0 and @Holzhaus I know this probably took sometime to fix, but here below is an image of the new `:kbd:` 😄 . I did override the initial style of `:kbd:` with new styles? What do you think? I can keep tweaking the styles until they match what you need.
![Uploading Mixxx3.PNG…]()

**Comment by hannydevelop** [comment]
Hello @ronso0 and @Holzhaus I know this probably took sometime to fix, but here below is an image of the new :kbd: 😄 . I did override the initial style of :kbd: with new styles? What do you think? I can keep tweaking the styles until they match what you need.
![Mixxx3](https://user-images.githubusercontent.com/62933980/97050495-608ad400-1575-11eb-8039-bab3fa258441.PNG)

**Comment by hannydevelop** [comment]
Should I add a touch of orange? (I guess it's Mixxx colour). I know the example above is too plain, I just need your suggestions.

**Comment by hannydevelop** [comment]
I think the style is cool though, especially from this picture
![Mixxx4](https://user-images.githubusercontent.com/62933980/97050889-122a0500-1576-11eb-8237-dfdd89813fcd.PNG)

**Comment by Holzhaus** [comment]
The two empty boxes on each side of the plus sign look weird, but other than that I really like it.

**Comment by hannydevelop** [comment]
> The two empty boxes on each side of the plus sign look weird, but other than that I really like it.

😆 I would make a PR to effect this design and would make sure the empty boxes are removed! Thanks for the review.

**Comment by ronso0** [comment]
> I think the style is cool though, especially from this picture
> ![Mixxx4](https://user-images.githubusercontent.com/62933980/97050889-122a0500-1576-11eb-8237-dfdd89813fcd.PNG)

Even though that is a nice idea I think it stands out too much from the surroundng text.
IMO the shadowed boxes you proposed are sufficient:
![Mixxx3](https://user-images.githubusercontent.com/62933980/97050495-608ad400-1575-11eb-8039-bab3fa258441.PNG)

**Comment by Holzhaus** [comment]
From the semantic perspective I quite like it and I think this is why sphinx support nested `:kbd:` roles: https://github.com/sphinx-doc/sphinx/issues/7530

Maybe it's possible to make the outer box Transparent if two kbd elements are nested? No sure if this is possible with CSS.

**Comment by hannydevelop** [comment]
Ok, I will try to make the outer box Transparent(If possible) and reduce font!

**Comment by Holzhaus** [comment]
Can you open a draft PR? Then we can already have a look you don't need to share Screenshots. :D

**Comment by ronso0** [comment]
don't confuse transparent with translucent (I assume @Holzhaus referred to the latter)

**Comment by hannydevelop** [comment]
Ok @ronso0
